src/app/store/subnet/types/base.ts

```diff
@@ -120,3 +125,14 @@ export type SubnetState = GenericState<Subnet, APIError> & {
   eventErrors: EventError<Subnet, APIError, SubnetMeta.PK>[];
   statuses: SubnetStatuses;
 };
+
+export type StaticDHCPLease = {
+  id: number;
+  comment: string | null;
+  ip_address: string;
+  mac_address: string;
+  interface: string;
+  node: SimpleNode;
+  usage: string | null;
+  owner: string;
+};
```

Comment:
`node`, `interface`, `usage` and `comment` should all be optional. A reservation can be made for a node that hasn't been commissioned yet, so that data isn't guaranteed.

Reply:
Let's just make sure all optional keys have `| null`, (`node?: SimpleNode | null;`) and not optional key (`node?: SimpleNode;`). With an optional key it can be unclear whether the property was omitted intentionally or by mistake. `null` states explicitly that a property has been intentionally omitted.